index.html

```diff
@@ -0,0 +1,95 @@
+    <link rel= "stylesheet" type="text/css" href= "style.css">
+</head>
+<a name="top"></a>
+<body>
+    <header class="navBar featureImage">
+        <div class="container">
+                <ul>
+                    <li><a href="#top" class="navDefault">HOME</a></li>
+                    <li><a href="#about">ABOUT ME</a></li>
+                    <li><a href="#feature">PORTFOLIO</a></li>
+                    <li><a href="#footer">CONTACT</a></li>
+                    <li><a href="#top" class="navHidden">HOME</a></li>
+                </ul>
+            <h1>RICHARD RABULAN</h1>
+            <p>Web Developer</p>
+            <a href="#feature" class="more">MORE</a>
+        </div>
+    </header>
+    <section class="portfolio">
+        <a id="feature" name="feature"></a>
+        <div class="boxes">
+            <h3>PORTFOLIO</h3>
+            <div class="projectBox project1 projectDone">
+                <h3>PROJECT 1</h3>
+            </div>
+            <div class="projectBox project2 projectDone">
+                <h3>PROJECT 2</h3>
+            </div>
+            <div class="projectBox project3">
+                <h4>PROJECT 3</h4>
+                <p>UNDER CONSTRUCTION</p>
+            </div>
+            <div class="projectBox project4">
+                <h4>PROJECT 4</h4>
+                <p>UNDER CONSTRUCTION</p>
+            </div>
+            <div class="projectBox project5">
+                <h4>PROJECT 5</h4>
+                <p>UNDER CONSTRUCTION</p>
+            </div>
+            <div class="projectBox project6">
+                <h4>PROJECT 6</h4>
+                <p>UNDER CONSTRUCTION</p>
+            </div>
+        </div>
+    </section>
+    <section class="aboutMe">
+        <a name="about"></a>
+        <div class="container">
+            <h2>ABOUT ME</h2>
+            <p>My name is Richard and I am aspiring to become a full stack developer. I am currently studying at the Mayden Academy in Bath and have been recently certified as a Scrum Master.</p>
+            <p>In addition to my growing web development skills, I have a Bachelor of Arts degree in Architecture and have been working as an Architectural Technician for the last 3 years. During that time I have acquired skills that are similar in nature to being a developer as well as soft skills that makes me an excellent addition to a team. In my free time, I enjoy photography, cooking and creating digital artworks using architectural software.</p>
+        </div>
+    </section>
+    <footer class="footer">
+        <a name="footer"></a>
+        <section class="contact">
+                <h2>CONTACT ME</h2>
+                <p><a href="tel:07708021930">Phone</a>
+                </p>
+                <p><a href="mailto:richard.rabulan@hotmail.co.uk">Email</a></p>
+        </section>
+        <div>
+            <a href="#top" class="backUp">Back to Top</a>
+        </div>
+        <div class="container">
+            <div class="logoContainer">
```

Comment:
not sure if you need both the container and logoContainer div, could probably just turn into one div

Reply:
container keeps the whole page at 90% width from middle, I'm afraid that if I turn it into one div that it could affect other <a>, <img> and would create more classes to keep the changes in footer from affecting other parts with container class